CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+## v0.6.0 (2017-01-28)
+
+#### Breaking change
+* `react-server-cli`, `react-server`
+  * [#837](https://github.com/redfin/react-server/pull/837) Address routr api changes & Handle POST. ([@drewpc](https://github.com/drewpc))
+* `react-server-cli`, `react-server`
```

Comment:
This heading wound up the same as the one above with the test packages removed.  Could just kill it and merge the groups.